src/Symfony/Component/HttpKernel/CacheWarmer/CacheWarmerAggregate.php

```diff
@@ -93,12 +95,17 @@ public function warmUp(string $cacheDir): array
                     continue;
                 }
 
+                $start = microtime(true);
                 foreach ((array) $warmer->warmUp($cacheDir) as $item) {
                     if (is_dir($item) || (str_starts_with($item, \dirname($cacheDir)) && !is_file($item))) {
                         throw new \LogicException(sprintf('"%s::warmUp()" should return a list of files or classes but "%s" is none of them.', $warmer::class, $item));
                     }
                     $preload[] = $item;
                 }
+
+                if ($io?->isDebug()) {
+                    $io->info(sprintf('"%s" completed in %0.2f ms.', $warmer::class, 1000 * (microtime(true) - $start)));
```

Comment:
I'd remove the space before ms (as in my previous suggestion ;) )

Reply:
Oops missed it 👀 That's updated as well!

Reply:
Is there some alternative syntax to write it?

```php
$io->info(sprintf('"%s" completed in %0.2fms.'
```

It's hard to read `%0.2fms` and figure out that `%0.2f` is a placeholder.

Reply:
What's important is what is displayed to end users, and this space is not the best we can do :)

Reply:
Of course, I was thinking from a maintainer's point of view.

I couldn't find a better way to write this so it looks like we have no other choice here.